src/lib/rest/rest.cpp

```diff
@@ -835,6 +834,14 @@ int servicePathSplit(ConnectionInfo* ciP)
     LM_I(("Service Path %d: '%s'", ix, ciP->servicePathV[ix].c_str()));
   }
 
+  //
+  // stringSplit destroys ciP->servicePath.
```

Comment:
This is obscure... instead of manipulating `ciP->httpHeaders.servicePath` after servicePathSplit() processing so it points to the first element to be prepared for add() to work, we should directly pass `ciP->httpHeaders.servicePath[0]` to add().